internal/platform/state/sync/evolution/sync.go

```diff
@@ -27,3 +33,49 @@ func (s *service) SetLastEvoBlock(block model.Block) error {
 func (s *service) GetLastEvoBlock() (model.Block, error) {
 	return sync.GetBlock(s.tx, lastBlock)
 }
+
+// SetNextEvoEventBlockForOwnershipContract is used by evo processor for storing the next block that has events
+func (s *service) SetNextEvoEventBlock(contract string, blockNumber uint64) error {
+	value, err := s.tx.Get([]byte(lastEvoEventBlockPrefix + strings.ToLower(contract)))
+	if err != nil {
+		return err
+	}
+
+	uintValue := uint64(0)
```

Comment:
I've put it nextValue

Reply:
this is actually just for converting `value` to uint. later it was again converted to the []byte. I have to do this check because if the value does not exist returns error, and I want it to be  0.

Reply:
considering that it is the same variable but just convert type I called it uintValue. I think that nextValue is not really a good option. Can we leave it as is?